### ISSUE TITLE
"Test lakeFS multipart upload with Hadoop S3A" is flaky

### ISSUE THREAD

**Issue by itaiad200** [treeverse/lakeFS#3686]
Example: https://github.com/treeverse/lakeFS/runs/7395107778?check_suite_focus=true#logs

Looks like it's always a network issue:
```
log4j:WARN See http://logging.apache.org/log4j/1.2/faq.html#noconfig for more info.
[error] (run-main-0) com.amazonaws.AmazonClientException: Unable to upload part: Unable to execute HTTP request: Broken pipe (Write failed)
[error] com.amazonaws.AmazonClientException: Unable to upload part: Unable to execute HTTP request: Broken pipe (Write failed)
[error] 	at com.amazonaws.services.s3.transfer.internal.UploadMonitor.collectPartETags(UploadMonitor.java:256)
[error] 	at com.amazonaws.services.s3.transfer.internal.UploadMonitor.completeMultipartUpload(UploadMonitor.java:236)
```

**Comment by arielshaqed** [comment]
Plan: start by skipping the test.  It is old and not particularly informative.

**Comment by arielshaqed** [comment]
> Plan: start by skipping the test. It is old and not particularly informative.

Updated: Reduce network access and total size.  This is not a stress test, we only need to exercise MPU to ensure that it works.

**Comment by arielshaqed** [comment]
_Should_ have been fixed by #3850.  Closing, but _please_ feel free to reopen if it happens again.
